test/keysym.c

```diff
@@ -27,6 +27,9 @@
 #include "test.h"
 #include "keysym.h" /* For unexported is_lower/upper/keypad() */
 
+#define STRINGIFY_(x) #x
```

Comment:
Maybe move this to utils.h?

Also, maybe change `STRINGIFY_` -> `STRINGIFY` and `STRINGIFY` -> `STRINGIFY2`? Sometimes you don't want to expand (e.g. the one in `x11/keymap.c` doesn't.

Reply:
Done